src/app-layout/visual-refresh-toolbar/toolbar/trigger-button/index.tsx

```diff
@@ -28,6 +29,50 @@ export interface TriggerButtonProps {
   onClick: React.MouseEventHandler<HTMLButtonElement>;
   badge?: boolean;
   highContrastHeader?: boolean;
+  /**
+   * If the button is expected to have a tooltip. When false it will not set the event listeners
+   *
+   * defaults to false
+   */
+  hasTooltip?: boolean;
+  /**
+   * This text allows for a customized tooltip.
+   *
+   * When falsy, the tooltip will parse the tooltip form the aria-lable
+   */
+  tooltipText?: string;
+  /**
+   * If the AppLayout has a drawer that is open
+   * Used to hide tooltips in certian conditions
+   */
+  /**
+   * set to true if the trigger button was used to open the last active drawer
+   */
+  isForPreviousActiveDrawer?: boolean;
```

Comment:
Just to make sure I understand, this is to prevent the tooltip from showing up again when it is focused after closing the drawer?

Could we add an explanation in the comment?

Reply:
Yes. Will do. this part of the logic will need to be revisited as it does not work when focus and hover events are combined via a bug bash finding.